evm/src/main/java/org/hyperledger/besu/evm/precompile/KZGPointEvalPrecompiledContract.java

```diff
@@ -37,52 +31,53 @@
 public class KZGPointEvalPrecompiledContract implements PrecompiledContract {
   private static final AtomicBoolean loaded = new AtomicBoolean(false);
 
-  private final Bytes successResult;
+  private static final Bytes successResult;
+
+  static {
+    CKZG4844JNI.loadNativeLibrary(CKZG4844JNI.Preset.MAINNET);
```

Comment:
Please move this initialization to a lazy or on-demand initialization.  

This will incur a large startup penalty on _every_ CLI call that touches the EVM.  This imposes a debilitating penalty on the EVM Tool and will make using Besu in reference tests impractical.

Reply:
make sense, will move it in the `init` methods